src/sentry/migrations/0238_fill_org_onboarding_tasks.py

```diff
@@ -0,0 +1,581 @@
+        Project = orm['sentry.Project']
+
+        queryset = Organization.objects.all()
+
+        for org in RangeQuerySetWrapperWithProgressBar(queryset):
+            if Project.objects.filter(organization=org).count() > 0:
+                OrganizationOnboardingTask.objects.create(
+                    organization=org,
+                    task=OrganizationOnboardingTask.FIRST_PROJECT,
+                    status=OnboardingTaskStatus.COMPLETE
+                )
+
+            projects = Project.objects.filter(organization=org).exclude(first_event=None).order_by('first_event')
+            project_count = projects.count()
+            if project_count > 0:
+                OrganizationOnboardingTask.objects.create(
+                    organization=org,
+                    task=OrganizationOnboardingTask.FIRST_EVENT,
+                    status=OnboardingTaskStatus.COMPLETE,
+                    project_id=projects[0].id
+                )
+
+                if project_count > 1:
+
+                    platforms = set()
+                    for project in projects:
+                        # Lazily assume project's events are uniformly of the same platform
+                        group = Group.objects.filter(project=project).first()
+                        if group:
+                            platforms.add(group.platform)
+                            if len(platforms) > 1:
+                                OrganizationOnboardingTask.objects.create(
+                                    organization=org,
+                                    task=OrganizationOnboardingTask.SECOND_PLATFORM,
+                                    status=OnboardingTaskStatus.COMPLETE,
+                                    project_id=project.id
+                                )
+                                break
+
+            # INVITE_MEMBER
+            if OrganizationMember.objects.filter(organization=org).exclude(user=None).count() > 1:
+                OrganizationOnboardingTask.objects.create(
+                    organization=org,
+                    task=OrganizationOnboardingTask.INVITE_MEMBER,
+                    status=OnboardingTaskStatus.COMPLETE,
+                )
+
+            # ISSUE_TRACKER
+            option = ProjectOption.objects.filter(
+                project__organization=org,
+                key__in=['github:enabled', 'pivotal:enabled', 'bitbucket:enabled', 'trello:enabled',
+                    'campfire:enabled', 'jira:enabled', 'sprintly:enabled', 'gitlab:enabled',
+                    'redmine:enabled', 'phabricator:enabled', 'taiga:enabled', 'lighthouse:enabled',
+                    'teamwork:enabled'],
+            ).first()
+            if option:
+                OrganizationOnboardingTask.objects.create(
+                    organization=org,
+                    task=OrganizationOnboardingTask.ISSUE_TRACKER,
+                    status=OnboardingTaskStatus.COMPLETE,
+                    project_id=option.project.id,
```

Comment:
`option.project_id` instead of `option.project.id`. The latter introduces another query.